### PR TITLE
Add multi-tenancy tutorial

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/learn/configuring-multi-tenancy.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Configuring Multi-Tenancy
+
+## Scenario
+
+Consider the following scenario.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: Not required

File: en/docs/learn/configuring-multi-tenancy.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Add users
+
+You can now log in to each tenant using the relevant tenant admin's credentials and create users and roles in the tenant. Note that you are managing two separate sets of users and roles for each tenant using one instance of WSO2 Identity Server.
+
+1. Log out of the Management Console and log in using the `pickup-eats` admin credentitals.
+
+    - **Username:** jane@pickup-eats.com
+    - **Password:** jane123
+
+2. Create a new user called "Cameron". For instructions on how to create a user, see [Adding Users and Roles](../../learn/adding-users-and-roles#create-a-user).
```
Comment (gomathyk) [inline en/docs/learn/configuring-multi-tenancy.md]: Shall we remove double quotes in "Cameron" and maybe use bold?

File: en/docs/learn/configuring-multi-tenancy.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Configuring Multi-Tenancy
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: Better to give a brief intro to multi-tenancy and point off to a page that discusses it detail.

File: en/docs/learn/configuring-multi-tenancy.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    - **Username:** jane@pickup-eats.com
+    - **Password:** jane123
+
+2. Create a new user called "Cameron". For instructions on how to create a user, see [Adding Users and Roles](../../learn/adding-users-and-roles#create-a-user). 
+
+You have succesfully created a user in the `pickup-eats` tenant. Now repeat step 1-2 and create a user in the `pickup-taxis` tenant as well. 
+
+## Try it out
+
+1. Log in to the [WSO2 Identity Server User Dashboard](https://localhost:9443/dashboard) using pickup-eats user credentials.
```
Comment (gomathyk) [inline en/docs/learn/configuring-multi-tenancy.md]: using the pickup-eats

File: en/docs/learn/configuring-multi-tenancy.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+A taxi company called "Pickup" is expanding into the food delivery business and opening a new segment called "PickupEats". Pickup now has two separate business models; one for PickupEats and one for PickupTaxis. Employees and customers will still be using the same Pickup application for both services, but Pickup wishes to manage their users in two isolated environments and have different settings/policies for each. 
+
+In order to do this, Pickup can set up two tenants that share access to the same application but are isolated, so that PickupEats and PickupTaxis can function as two separate segments. This concept is called **multi-tenancy**.  This tutorial demonstrates multi-tenancy in WSO2 Identity Server. 
+
+## Create a tenant
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` and start the server by executing one of the following commands.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: bin directory via a command prompt

File: en/docs/learn/configuring-multi-tenancy.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+5. Enter the following tenant details.
+    1. **Domain**: pickup-eats.com
+    2. **Usage plan for the tenant** - Demo
+    3. **First Name** - Jane
+    4. **Last Name** - Doe
+    5. **Admin Username** - jane@pickup-eats.com
+    6. **Admin Password** - jane123
+    7. **Email** - janedoe@gmail.com
+    
+You have now successfully created a tenant for the PickupEats. Repeat steps 4-5 and create a tenant called "pickup-taxis.com" with different admin credentials.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: the first sentence can be part of the 5th instruction and the second sentence can be the 6th sentence- to create another tenant called "", repeat the ...

File: en/docs/learn/configuring-multi-tenancy.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    4. **Last Name** - Doe
+    5. **Admin Username** - jane@pickup-eats.com
+    6. **Admin Password** - jane123
+    7. **Email** - janedoe@gmail.com
+    
+You have now successfully created a tenant for the PickupEats. Repeat steps 4-5 and create a tenant called "pickup-taxis.com" with different admin credentials. 
+
+## Add users
+
+You can now log in to each tenant using the relevant tenant admin's credentials and create users and roles in the tenant. Note that you are managing two separate sets of users and roles for each tenant using one instance of WSO2 Identity Server.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: ideally should be part of the previous section.

File: en/docs/learn/configuring-multi-tenancy.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Log out of the Management Console and log in using the `pickup-eats` admin credentitals.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: Sign in

Comment (YvonneW) [reply en/docs/learn/configuring-multi-tenancy.md]: Sign out

File: en/docs/learn/configuring-multi-tenancy.md (hunk 3 above)
Comment (YvonneW) [reply en/docs/learn/configuring-multi-tenancy.md]: All Field values should be in monospace

File: en/docs/learn/configuring-multi-tenancy.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+    - **Username:** cameron@pickup-eats.com
+    - **Password:** cameron123
+
+2. Next, log out and log in using the `pickup-taxis` user credentials. 
+
+    - **Username:** alex@pickup-taxis.com
+    - **Password:** alex123
+
+Note the relevant tenant domain appended to the username of each user. You have successfully configured two separate tenants for PickupEats and PickupTaxis with shared access to the dashboard application.
```
Comment (YvonneW) [inline en/docs/learn/configuring-multi-tenancy.md]: user name

File: en/docs/learn/configuring-multi-tenancy.md (hunk 8 above)
Comment (YvonneW) [reply en/docs/learn/configuring-multi-tenancy.md]: As Management Console is a  UI console not an application.

**Pull request merged**